src/RightComponent.svelte

```diff
@@ -0,0 +1,34 @@
+<div class="rightComponentContainer">
+    <h1>Download the Uptown app</h1>
+    <p>View vendors</p>
+    <p>View Turn ups</p>
+    <p class="lastOne">Manage your business efficiently</p>
+
+    <a target="_blank" href="https://google.com">
+<!--        <img style="height: 50px;" src="https://media.playstation.com/is/image/SCEA/playstation-app-app-store-badge-image-block-01-ps4-us-01aug19?$native_nt$" />-->
+        <img style="height: 50px; width: 170px" src="coming soon.jpeg" />
```

Comment:
Please do not use spaces with images. 
You can use either camelCase (comingSoon)
or underscore (coming_soon)